src/PIL/ImageGrab.py

```diff
@@ -61,10 +61,13 @@ def grab(bbox=None, include_layered_windows=False, all_screens=False, xdisplay=N
                 left, top, right, bottom = bbox
                 im = im.crop((left - x0, top - y0, right - x0, bottom - y0))
             return im
-        elif shutil.which("gnome-screenshot"):
+        elif shutil.which("gnome-screenshot") or shutil.which("scrot"):
```

Comment:
For clarity, I would put the two options in the order they are used in:
```suggestion
        elif shutil.which("scrot") or shutil.which("gnome-screenshot"):
```